tests/test_autonomy/test_cli/test_deploy/test_from_token.py

```diff
@@ -52,7 +51,13 @@ def test_from_token(
         self,
     ) -> None:
         """Run test."""
-
+        mock_ipfs_response = {
+            "name": "valory/oracle_hardhat",
+            "description": "Oracle service.",
+            "code_uri": "ipfs://bafybeiansmhkoovd6jlnyurm2w4qzhpmi43gxlyenq33ioovy2rh4gziji",
+            "image": "bafybeiansmhkoovd6jlnyurm2w4qzhpmi43gxlyenq33ioovy2rh4gziji",
+            "attributes": [{"trait_type": "version", "value": "0.1.0"}],
+        }
```

Comment:
this is the response from an actual request the command makes

Reply:
The problem is this will end up outdated

Reply:
>The problem is this will end up outdated

In a way this is not a problem, since we just want to check if the on-chain token resolution is working or not.  I think we can introduce integration tests where we actually interact with external services and keep this as an unit test.

Reply:
I just realized that there is a ipfs download going on after this ipfs response is returned. I assume you mean that it will get outdated because we download from ipfs with that specific hash. 
What if we just mock the download to copy from the local service package instead of actually trying to download form ipfs?

Reply:
This is an e2e test, so we should use the external dependency imo. The mocking will just get out of sync and be hard to maintain.

Reply:
Okay, so we have to "fixes" in this PR.
1. Increased max retries.
2. Mocking of ipfs req.

I can revert 2, but the test will remain flaky.

Reply:
Yes let's revert 2 for now and instead increase retries (done) and also add flaky test marker so it reruns the test on failure